packages/client/src/pages/Login/Login.tsx

```diff
@@ -89,7 +92,7 @@ class Login extends React.Component<LoginProps, LoginState> {
 
     render() {
         // Return the user back to the page before login
-        const { from } = this.props.location.state || { from: { pathname: '/' } };
+        const { from } = this.props.location.state || { from: { pathname: '/mytrecipes' } };
```

Comment:
We set `this.props.location.state` in `PrivateRoute.tsx` whenever the user visit a private route. For example, let say our map page is private, 
1) User visit the map page without signing in,  we update the state to store map page url and redirect them back to login page
2) On login page, user tried to sign in
3) Once they are signed in, we will can use the state to get the url and redirect them back to map page

So `this.props.location.state` is undefined when we visit login page from a public route. And this is pretty useless right now since the only private route is mytrecipe page lol

Reply:
Ahh i see. Thanks for explaning!